docs/theory/heat_physics.rst

```diff
@@ -0,0 +1,372 @@
+
+:math:`\dot{Q}^{a}_{consumed}` can be defined by the thermal power substracted from or added to the network, where the internal losses of the storage are subtracted:
+
+.. math::
+    :label: eq:change_stored_heat
+
+    \dot{Q}^{a}_{consumed} =  \sum_{i \in I^a_{in}} \dot{Q}^{a}_{i} -  \sum_{i \in I^a_{out}} \dot{Q}^{a}_{i} - \dot{Q}^{a}_{loss} \;\; \forall a \in A_{storage}.
+
+The consumed heat of the storage assets is equated to the change change in stored heat, :math:`\dot{Q}^{a}_{stored}`:
+
+.. math::
+    :label: eq:stored_heat
+
+    \dot{Q}^{a}_{consumed} = \dot{Q}^{a}_{stored} \;\; \forall a \in A_{storage}
+
+Like the source and demand assets the storage needs equality constraints relating the outgoing flow to the thermal power.
+Unlike the demand and source the outgoing flow can be either on the in or out port depending whether the storage is charging or discharging.
+The default convention is that charging indicates positive flow, therefore :math:`\delta_{dir}` can be used as an integer for charging/discharging.
+Equations :eq:`eq:storage_heat2discharge1` and :eq:`eq:storage_heat2discharge2` are active during charging, and during discharging,
+:eq:`eq:storage_heat2discharge3` and :eq:`eq:storage_heat2discharge4` are activated by the big M method.
+
+.. math::
+    :label: eq:storage_heat2discharge1
+
+     \sum_{i \in I^a_{in}} \dot{Q}^{a}_{i} - c_p \rho \dot{V}^{a} T_{sup} \leq 0 \;\; \forall a \in A_{storage}
+
+.. math::
+    :label: eq:storage_heat2discharge2
+
+     \sum_{i \in I^a_{in}} \dot{Q}^{a}_{i} - c_p \rho \dot{V}^{a} T_{sup} + \delta^{a}_{dir} M \geq 0 \;\; \forall a \in A_{storage}
+
+.. math::
+    :label: eq:storage_heat2discharge3
+
+     \sum_{i \in I^a_{out}} \dot{Q}^{a}_{i} - c_p \rho \dot{V}^{a} T_{ret} \geq 0 \;\; \forall a \in A_{storage}
+
+.. math::
+    :label: eq:storage_heat2discharge4
+
+     \sum_{i \in I^a_{out}} \dot{Q}^{a}_{i} - c_p \rho \dot{V}^{a} T_{ret} - (1-\delta^{a}_{dir}) M \leq 0 \;\; \forall a \in A_{storage}
+
+Tank
+^^^^
+
+Heat loss in a tank is modelled as linear with the stored heat:
+
+.. math::
+    :label: eq:storage_loss
+
+    \dot{Q}^{a}_{loss} = \beta^{a} Q^{a}_{stored} \;\; \forall a \in A_{storage},
+
+where :math:`Q^{a}_{stored}` is the heat stored in the storage asset and :math:`\beta` is the efficiency factor.
+
+The efficiency factor is approximated assuming that tanks are cylindrical and lose heat over their surface area, see :eq:`eq:etatank`.
+For cylindrical tanks their surface area approximately increases linearly with the stored heat.
+A radiation coefficient, :math:`c_r`, of 1 :math:`W/m^2` is used as an approximation.
+
+.. math::
+    :label: eq:etatank
+
+    \beta_{tank} = \frac{2c_r}{r\rho c_p}
```

Comment:
maybe remove this part as in the paper, because it might seem the beta is a variable and thus also changing when sizing

Reply:
The equation has been removed